packages/react/lib/template.js

```diff
@@ -2,35 +2,41 @@
 
 const esc = require('serialize-javascript');
 
-const renderCSS = ({ css }) =>
+const renderCSS = (css) =>
   css.map((asset) => `<link rel="stylesheet" href="/${asset}" />`).join('');
 
-const renderJS = ({ js }) =>
+const renderJS = (js) =>
   js.map((asset) => `<script src="/${asset}"></script>`).join('');
 
 const renderGlobals = (globals) => {
   const entries = Object.entries(globals).map(([k, v]) => `${k}=${esc(v)}`);
   return entries.length ? `<script>var ${entries.join(',')};</script>` : '';
 };
 
-module.exports = ({ fragments, assets, mountpoint, markup, globals }) =>
+module.exports = ({
+  markup,
+  mountpoint,
+  fragments = {},
+  globals = {},
+  assets: { css = [], js = [] } = {},
+}) =>
   `<!DOCTYPE html>
-<html ${fragments.htmlAttributes}>
+<html ${fragments.htmlAttributes || ''}>
   <head>
-    ${fragments.headPrefix}
-    ${fragments.title}
-    ${fragments.base}
-    ${fragments.meta}
-    ${fragments.link}
-    ${renderCSS(assets)}
-    ${fragments.style}
-    ${fragments.script}
-    ${fragments.headSuffix}
+    ${fragments.headPrefix || ''}
+    ${fragments.title || ''}
+    ${fragments.base || ''}
+    ${fragments.meta || ''}
+    ${fragments.link || ''}
+    ${renderCSS(css)}
+    ${fragments.style || ''}
+    ${fragments.script || ''}
+    ${fragments.headSuffix || ''}
   </head>
-  <body ${fragments.bodyAttributes}>
-    <div id="${mountpoint}">${markup}</div>
-    ${fragments.noscript}
+  <body ${fragments.bodyAttributes || ''}>
+    <div id="${mountpoint || ''}">${markup || ''}</div>
```

Comment:
Since we are using `config.name` to determine the mountpoint, I personally do not think we have to handle this case specifically... See:

https://github.com/untool/untool/blob/master/packages/react/lib/render.js#L43

Reply:
We could of cause add a safeguard later on.